### ISSUE TITLE
Add support for new metric: Mean Absolute Percentage Error

### ISSUE THREAD

**Issue by pplonski** [mljar/mljar-supervised#310]
I will use metric from scikit-learn https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_absolute_percentage_error.html

**Comment by pplonski** [comment]
Hi @girardea, I've added support to the MAPE metric. The code update is in the `dev` branch. To install the newest code please run:

```
pip install -U git+https://github.com/mljar/mljar-supervised.git@dev
```

To use MAPE metric please set `eval_metric="mape"`, like below:

```python
automl = AutoML(eval_metric="mape")
automl.fit(X, y)
```

The code updates will go to the `0.9.0` version.

**Comment by girardea** [comment]
That's great news! Will test it right away!
